### ISSUE TITLE
Deployment scripts

### ISSUE THREAD

**Issue by smburdick** [Puget-Sound-Bitmaps/distributed-bitmap-engine#2]
I think it'd be nice to have deployment scripts to automate the code shipment process.
Possibilities:
- Bootstrapping script in separate repo, to initiate the entire process
- We could initialize the whole process on master node, which `sftp`s all the code to the given slave nodes (the issue is that the slave processes need to be started somehow)
-- Or, just initialize on each node (let the slaves just wait on the master)
